apps/payments/next/app/[locale]/subscriptions/landing/route.ts

```diff
@@ -47,7 +47,7 @@ export async function GET(
         redirect: false,
         redirectTo: redirectToUrl.href,
       },
-      { prompt: 'none', return_on_error: 'false' }
+      { prompt: 'login' }
```

Comment:
question (blocking): Is this change necessary?

The previous code had it set to prompt=none with login as a fallback if it fails, which seems more desirable than always forcing login.